src/dscim/menu/main_recipe.py

```diff
@@ -1135,9 +1139,17 @@ def uncollapsed_sccs(self):
 
         return sccs
 
-    # @cachedproperty
-    # def quantiles_sccs(self):
-    #     return self.uncollapsed_sccs.quantile(self.scc_quantiles, dim=self.fair_dims)
+    @cachedproperty
+    @save(name="stat_uncertainty_iqr")
+    def stat_uncertainty_iqr(self):
+        """Calculate the distribution of quantile-weighted SCCs produced from
+        quantile regressions collapsed across pulse dimension.
```

Comment:
"collapsed across pulse dimension" is a bit confusing. This IQR is taken across only the quantreg quantile dimension because there is no climate uncertainty dimension. This is trickier with RFF-SPs and we need to test it.

Reply:
Let's reword "quantile regressions that have already been collapsed across other dimensions to give statistical-only uncertainty". 
In a separate PR, we should nicely fail if someone tries to run quantreg with RFF-SPs.